proto/babylon/btcstkconsumer/v1/btcstkconsumer.proto

```diff
@@ -1,15 +1,49 @@
 syntax = "proto3";
 package babylon.btcstkconsumer.v1;
 
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+
 option go_package = "github.com/babylonlabs-io/babylon/x/btcstkconsumer/types";
 
+// ConsumerType defines the type of consumer chain
+enum ConsumerType {
+  // COSMOS represents a Cosmos SDK chain consumer
+  COSMOS = 0;
+  // ETH_L2 represents an Ethereum L2 chain consumer
+  ETH_L2 = 1;
+}
+
 // ConsumerRegister is the registration information of a consumer
 message ConsumerRegister {
   // consumer_id is the ID of the consumer
   // - for Cosmos SDK chains, the consumer ID will be the IBC client ID
+  // - for ETH L2 chains, the consumer ID will be the chain ID of the ETH L2
+  //   chain
   string consumer_id = 1;
```

Comment:
Nit: What about adding this field to both metadatas below, with a proper description for each?

Reply:
I would prefer having it here because every consumer, regardless of its type, will have a consumer ID. Wdyt?